packages/cozy-konnector-libs/src/libs/saveBankingDocuments.js

```diff
@@ -41,20 +45,32 @@ module.exports = (entries, fields, options = {}) => {
     fields = { folderPath: fields }
   }
 
+  options.doctype = options.doctype || DEFAULT_DOCTYPE
+
   // Deduplicate on this keys
   options.keys = options.keys || ['date', 'amount', 'vendor']
 
   options.postProcess = function (entry) {
     if (entry.fileDocument) {
-      entry.invoice = `io.cozy.files:${entry.fileDocument._id}`
+      entry.$relationships = entry.$relationships || {}
+      entry.$relationships.document = {
```

Comment:
Is `$relationships` a cheerio object? Then couldn't it happen that such a common name as `document` would make its way into cheerio's API?

Reply:
`$relationships` is not a cheerio object. It is a cozy-special fields for relationships (see https://github.com/cozy/cozy-doctypes/pull/38)